test/memspaces/memspace_highest_capacity.cpp

```diff
@@ -40,28 +41,27 @@ TEST_F(memspaceHighestCapacityProviderTest, highestCapacityVerify) {
     static constexpr size_t alloc_size = 1024;
 
     long long maxCapacity = 0;
-    int maxCapacityNode = -1;
+    std::vector<int> maxCapacityNodes{};
     for (auto nodeId : nodeIds) {
         if (numa_node_size64(nodeId, nullptr) > maxCapacity) {
-            maxCapacityNode = nodeId;
             maxCapacity = numa_node_size64(nodeId, nullptr);
         }
+
+        if (numa_node_size64(nodeId, nullptr) == maxCapacity) {
```

Comment:
shouldn't this `if` be in a sep. `for`...?

Reply:
Yes, of course, done.